src/components/side-panel/layer-panel/layer-type-selector.tsx

```diff
@@ -56,10 +82,10 @@ function LayerTypeSelectorFactory(LayerTypeListItem, LayerTypeDropdownList) {
       [hasData, layerTypeOptions]
     );
 
-    const selectedItems = useMemo(() => typeOptions.find(op => op.id === layer.type), [
-      typeOptions,
-      layer.type
-    ]);
+    const selectedItems = useMemo(() => {
+      const selected = typeOptions.find(op => op.id === layer.type);
+      return selected ? [selected] : [];
```

Comment:
selectedItems doesn't have to be an array

Reply:
fixed